challenges.tex

```diff
@@ -0,0 +1,540 @@
+		\node at (0.5,4.5) [circle,fill=magenta, draw=black,inner sep=0.2cm] {};
+		\node at (1.5,4.5) [rectangle,fill=green!30!white, draw=black,inner sep=0.1pt,align=center,font=\tiny,rotate=90] {input\\output};
+
+\end{tikzpicture}
+\caption{Starting configuration for the grasping challenge.
+	BS is placed at $\texttt{M\_Z12}$, CS1 is placed at $\texttt{M\_Z53}$
+and RS1 is placed at $\texttt{M\_Z25}$.}
+	\label{fig:grasping-field}
+\end{figure}
+
+\subsubsection{Product Challenges}\label{sec:challenge-cx}
+This section covers four types of challenges, which all can be individually
+completed.
+Each challenge corresponds to the production of a product with one of the
+available complexities (C0, C1, C2, C3) in the \ac{RCLL} using either
+one or two \ac{RS}.\\
+For complexities C1, C2 and C3 the accumulated cost for mounting the required
+rings must be equal to 1, 2 and 3, respectively.
+\textbf{Task:} Produce all posted orders.\\
+\begin{table}[!htb]
+ \centering
+ \begin{tabular}{l|l|l}
+  Machines & Challenge type & Scoring \\\hline
+  2 & C0 & 30 \\
+  3 & C1 & 50 \\
+  4 & C1 & 50 \\
+  3 & C2 & 70 \\
+  4 & C2 & 70 \\
+  3 & C3 & 100 \\
+  4 & C3 & 100 \\
+ \end{tabular}
+ \caption{CX Challenge}
+ \label{tab:challenge-cx}
+\end{table}
+
+\subsubsection{Exploration + Production Challenges}\label{sec:challenge-combine-exp-c0}
+The same challenge as in \refsec{sec:challenge-cx} but without receiving the
+machine positions. Hence exploration and production takes place concurrently.
+The same point scores as in \reffig{tab:challenge-cx} apply with an additional
+$+20$ per challenge.
+%\subsubsection{RefBox Simulation Challenge}\label{sec:challenge-simulation}
+%A competition on the agent level. The RefBox provides a set of actions
+%that can be executed by sending dedicated commands to the RefBox via protobuf.
+%Hence no actual robot is required to participate.
+%\textbf{Task:} Play a regular RCLL game through the RefBox simulation
+%interface.\\
+%Participating in this challenge yields points based on the achieved in-game
+%points, see \reftab{tab:challenge-simulation}.
+%Additionally, the team scoring the highest points overall gets awarded another
+%$10$ points
+%\begin{table}[!htb]
+% \centering
+% \begin{tabular}{l|l|l|l}
+%  Points & Scoring \\\hline
+%  $[0,50)$ & 0\\% chktex 9
+%  $[50,150)$ & 20\\% chktex 9
+%  $[150,250)$ & 40 \\% chktex 9
+%  $[250,\infty)$ & 60 \\% chktex 9
+% \end{tabular}
+% \caption{CX Challenge}
+% \label{tab:challenge-cx}
+%\end{table}
+
+
+\subsubsection{Markerless Detection Challenge}\label{sec:markerless}
+Image recognition challenge to classify different machine types.\\
+\textbf{Task:} Autonomously label the machines shown in a set of pictures
+by their type.
+A picture may contain more than one machine, which all count towards the
+total number of machines to detect.\\
+As a preparation for this challenge, a data set will be supplied to
+all participants which may be used for training and testing purposes
```

Comment:
Till now I thought that it is allowed to train only with the supplied data set. Since most of the teams (including GRIPS) do not have access to real machines to create training data I think this is a quite large handicap in this challange if other teams can easily create more training data. How are you seeing this?

Reply:
I totally agree, the idea was to leave this open so teams can come up with additional creative ways to generate data (solidus thought about using images from the simulation).
At Carologistics we are aware of the advantage we have with access to the machines, which is why we only used the published data set for training.
Additionally, I don't know how big the contribution of other data might be because the images for the challenge will be taken from the same machines as the ones in the data set.